src/remotecontrol.cpp

```diff
@@ -72,32 +72,45 @@ void RemoteControl::handle()
     lastResult = result;
 
     auto &effectManager = g_ptrSystem->EffectManager();
+    auto& deviceConfig = g_ptrSystem->DeviceConfig();

```

Comment:
Now that the declaration of `deviceConfig` is immediately under `effectManager`, I think it's best to align the `auto` and the `&` to it:
```suggestion
    auto &deviceConfig = g_ptrSystem->DeviceConfig();
```

Reply:
Hey, all I did was cut and paste someone else's line of code and didn't notice the inconsistency. Maybe the person who originally typed it should have respected the style already implemented in the file... :)  Honestly I didn't notice when I pasted the line, but now that you pointed it out, it make my eye twitch. One thing I can say is there is no consistency in where to put the `&` in this codebase. I prefer to put it by the variable name, but others like to attach it to the type declaration. I am here to learn whichever way is "right."

Reply:
Both styles are used in the codebase and that's ok, until two lines are so close to each other that it looks weird to use one in one line and the other in another. Which I think is clearly the case when one line is line 74 and the other 75. :)  
It only stands out now because the deviceConfig declaration got moved a whole lot higher than it was.